src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+$size: 100px;
+$month-length: 31;
+$sun: 606px;
+
+.calendar {
+  gap: 1px;
+  display: flex;
+  box-sizing: border-box;
+  width: 726px;
```

Comment:
It's not a good practice to use absolute units for width. Consider using relative units like percentages or viewport units. This will make your layout more responsive.